zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/cluster/TestClusterBuilder.java

```diff
@@ -352,6 +352,12 @@ private TestStandaloneBroker createBroker(final int index) {
                   cluster.setClusterSize(brokersCount);
                   cluster.setClusterName(name);
                 })
+            .withBrokerConfig(
+                cfg -> {
+                  final var replicas = (partitionsCount * replicationFactor) / brokersCount;
+                  cfg.getThreads().setIoThreadCount(replicas);
```

Comment:
:thought_balloon: I'm wondering if this might be too many cpu threads as the total number of cpu threads will be probably higher than the machine core count. 
However, they are also completely independent so :shrug:

Reply:
Yeah I think there's no one size fits all solution but it's probably better to have a couple too many vs. too few.